internal/provider/provider.go

```diff
@@ -236,20 +242,6 @@ func New(version string) func() provider.Provider {
 	}
 }
 
-func setGlobalAnnotation(attribute basetypes.StringValue, envKey string) {
```

Comment:
I think this code may have been reverted too but pulling in the latest master and regenerating should fix this too.

Reply:
@anvitha-jain I think Akini has fixed it in this PR https://github.com/CiscoDevNet/terraform-provider-aci/pull/1167.
Would it be okay to revert these back for now?

Reply:
Not sure if this one has been fixed yet. The code still seems to be changed in `provider.go`. 
You may need to manually undo this code change.

Reply:
changes on provider.go file should also be reverted, or wait and rebase when #1167 is merged

Reply:
I merged #1167 so you should be able to rebase now.